paddle/fluid/framework/operator.h

```diff
@@ -225,8 +225,7 @@ class ExecutionContext {
     std::vector<const Variable*> res;
```

Comment:
Here, it would be even simpler with:
```
- std::vector<const Variable*> res;
- res.reserve(it->second.size());
- std::transform(it->second.begin(), it->second.end(),
-                  std::back_inserter(res), [](Variable* var) { return var; });
- return res;

+ return {it->second.begin(), it->second.end()};
```

Reply:
Exactly, I think I will just do that.